assets/aml-benchmark/components/inference-postprocessor/spec.yaml

```diff
@@ -29,10 +29,16 @@ inputs:
       Key in ground truth dataset that contains ground truth. If ground_truth_dataset is given, then,
       this is required input.
     optional: True
-  prediction_probabilities_dataset:
-    type: uri_file
+  remove_prefixes:
+    type: string
     description: >-
-      A file that contains the prediction probabilities.
+      A set of string prefixes separated by comma list of string prefixes to be removed from the inference
```

Comment:
@arun-rajora No, because when we say set of prefixes, it could mean the set of characters as we also pass the set of characters in " " and strip remove any of these characters that string starts with, and after removal it checks again if it starts with one of the characters given in set. Whereas string prefix operate differently. For example : "###,>>>" prefixes will first check if  string starts with '###', if yes, then it will be removed and then only the second given prefix is applied. That's the reason that you will find that we apply strip method while stripping characters but we don't apply this method while removing prefixes. I added word string while having a discussion with Sasha where we came to conclusion that prefixes field was there for strings.